docs/data/material/migration/migrating-from-deprecated-apis/migrating-from-deprecated-apis.md

```diff
@@ -227,6 +227,14 @@ Here's how to migrate:
  },
 ```
 
+## Autocomplete
+
+Use the [codemod](https://github.com/mui/material-ui/tree/HEAD/packages/mui-codemod#autocomplete-props) below to migrate the code as described in the following sections:
+
+```bash
+npx @mui/codemod@latest deprecations/autocomplete-props <path>
+```
+
```

Comment:
May I ask you to add a brief explanation for the deprecated props here? We can have one section for all, for example: https://github.com/mui/material-ui/pull/42062/files#diff-a5a0a00a518352d06d4ddc3013a1da70dcae1423ee73e024aae5649982f84e7eR1024